.github/workflows/quickstart.yaml

```diff
@@ -1,10 +1,9 @@
 name: Quickstart
 
 on:
-  push:
-    branches: main
-  pull_request:
-    branches: main
+  schedule:
+    # run at noon every day
+    - cron: "0 12 * * *"
```

Comment:
If we remove the pull_request part, then we can't test that it runs before we merge this and it hits the cron. Notice that the workflow hasn't run.